reps/2022-04-07-serialization.md

```diff
@@ -0,0 +1,162 @@
+## Summary
+### General Motivation
+Current Ray's serialization has some issues:
+
+1. Doesn't support out-of-band serialization in Java worker. Cause some perfomance issues in Java worker.
+1. Hard to add a new serializer for a specific class.
```

Comment:
class or format? class here is something like user defined or third party class; format is the serialized bytes format

Reply:
I think class makes more sense here. From the users' view, they want to add a serializer for a specific `class`. Or maybe `type`?